Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Management/API/Driver.yaml

```diff
@@ -0,0 +1,385 @@
+imports:
+  Driver: Dashboard.Common
+  Summary: Dashboard.Common
+  DriverIds: Dashboard.Common.Driver
+  Ride: Dashboard.Common
+  Image: Dashboard.Common
+  Category: Dashboard.Common
+  Variant: Dashboard.Common
+  DriverGoHomeRequest: Dashboard.Common
+  DriverHomeLocation: Dashboard.Common
+  VerificationStatus: Dashboard.Common
+  DriverDocumentsInfoRes: Dashboard.Common.Driver
+  DriverActivityRes: Dashboard.Common.Driver
+  DriverLicense: Dashboard.Common.Driver
+  VehicleRegistrationCertificate: Dashboard.Common.Driver
+  ReasonForDisablingServiceCharge: Dashboard.Common.Driver
+  ServiceNames: Dashboard.Common.Driver
+  Version: Kernel.Types.Version
+  LatLong: Kernel.External.Maps.Types
+  City: Kernel.Types.Beckn.Context
+
+module: Driver
+
+apis:
+  - GET: # DriverDocumentsInfoAPI
+      endpoint: /documents/info
+      auth: ApiAuth DRIVER_OFFER_BPP_MANAGEMENT DRIVERS DOCUMENTS_INFO
```

Comment:
TODO api name `DOCUMENTS_INFO`  and migration for dev api access also can be autogenerated